chain/api/src/lib.rs

```diff
@@ -350,7 +351,7 @@ impl<T: HoprDbAllOperations + Send + Sync + Clone + std::fmt::Debug + 'static> H
         Ok(self.rpc_operations.get_eligibility_status(self.me_onchain()).await?)
     }
 
-    pub async fn get_minimum_winning_probability(&self) -> errors::Result<f64> {
+    pub async fn get_minimum_winning_probability(&self) -> errors::Result<WinningProbability> {
```

Comment:
But why `WinningProbability`, when it should only be `Probability`?

Reply:
General probability does not have this weird encoding of ours. So that's why this is rather `WinningProbability`.